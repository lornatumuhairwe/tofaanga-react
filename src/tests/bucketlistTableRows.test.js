import React from 'react';
import renderer from 'react-test-renderer';
import BLRow from "../components/Bucketlist/bucketlistTableRows";

describe('bucketlist row snapshot', () => {
    it('matches snapshot', () => {
            const blrow = renderer.create(<BLRow/>);
            expect(blrow.toJSON()).toMatchSnapshot();
        },

    );
});
