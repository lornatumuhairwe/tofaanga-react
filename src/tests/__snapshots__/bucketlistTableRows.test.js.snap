// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bucketlist row snapshot matches snapshot 1`] = `
<li
  className="list-group-item col-md-12"
>
  <a
    onClick={[Function]}
  />
  <div
    className="pull-right"
  >
    <form
      className="form-inline"
    >
      <input
        name="name"
        required={true}
        type="hidden"
        value={undefined}
      />
      <button
        className="btn btn-xs"
        onClick={[Function]}
        type="submit"
      >
        <i
          className="glyphicon glyphicon-plus"
        />
      </button>
      <button
        className="btn btn-xs"
        onClick={[Function]}
        type="submit"
      >
        <i
          className="glyphicon glyphicon-edit"
        />
      </button>
      <button
        className="btn btn-xs"
        onClick={[Function]}
        type="submit"
      >
        <i
          className="glyphicon glyphicon-trash"
        />
      </button>
    </form>
  </div>
  <div
    className="notifications-wrapper"
    style={Object {}}
  />
  <div
    className="panel"
  >
    <div
      className="panel panel-info"
      id={null}
    >
      <div
        aria-expanded={null}
        aria-hidden={true}
        className="panel-collapse collapse"
        id={undefined}
        role={undefined}
      >
        <div
          className="panel-body"
        >
          <table
            className="table"
          >
            <thead>
              <tr>
                <th>
                  Item
                </th>
                <th>
                  Deadline
                </th>
                <th>
                  Status
                </th>
              </tr>
            </thead>
            <tbody />
          </table>
        </div>
      </div>
    </div>
  </div>
</li>
`;
